### ISSUE TITLE
does not support msgpack timestamp-extension-type

### ISSUE THREAD

**Issue by kb0thn** [ludocode/msgpack-tools#7]
Support for the timestamp extension would be great.
https://github.com/msgpack/msgpack/blob/master/spec.md#timestamp-extension-type

Of course the format for the JSON side needs to be decided. My preference would ISO 8601 format.

**Comment by ludocode** [comment]
Indeed! I am currently adding timestamp support to mpack in the [timestamp branch](https://github.com/ludocode/mpack/tree/timestamp). Once I get a decent battery of unit tests I'll start working on timestamp support here.

The JSON side will certainly use ISO 8601. It will take a bit of work because I haven't really found a decent/full-featured ISO 8601 parser for C/C++, and portable date conversions might cause some problems. It should be doable without too much trouble though.

**Comment by cielavenir** [comment]
hello, any news on this? seems msgpack-python have several options how to decode timestamp fields.
